assets/javascripts/modules/analytics/omniture.js

```diff
@@ -51,6 +53,18 @@ define([
         s.eVar19 = 'D=c19';
         s.prop42 = 'Subscriber';
         s.prop47 = 'UK';
+
+        var main = $('main')[0];
```

Comment:
Can we do this in a different way? It's completely by chance that we are using the `<main>` element for this, probably at least add a `.js-` class.

Couple of minor points: If you are just plucking the element out with `[0]` there's no need to use `$` and we could just use `querySelector` directly. The `map` could be a `forEach` as we're not actually returning anything (i.e., performing side-effects) nor using the result.

This code feels a bit magic generally though.